README.md

```diff
@@ -0,0 +1,85 @@
+# Safaa
+
+Safaa is a Python package designed for handling false positive detection in copyright notices. Additionally, it can declutter copytright notices, removing unnecessary extra text.
```

Comment:
```suggestion
Safaa is a Python package designed for handling false positive detection in copyright notices. Additionally, it can declutter copyright notices, removing unnecessary extra text.
```